content/d3/concepts/styling/terms/text/text.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '.text()'
+Description: 'This method is used to set or get text content of selected elements'
+Subjects:
+  - 'Data Visualization'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Data'
+  - 'Graphs'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+The **`.text()`** method in D3 is used to set text content for any selected element in a document, this method can also be used to get the value of any selected element. This method can take a single value as an argument or a callback function for a complex text content manipulation of a selected element
```

Comment:
```suggestion
The **`.text()`** method in D3 is used to set text content for any selected element in a document, it can also be used to get the value of any selected element. This method can take a single value as an argument or a callback function for a complex text content manipulation of a selected element
```
Use of same words is making the sentence sound wierd.